src/site/content/en/fast/measure-performance-with-the-rail-model/index.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: Measure performance with the RAIL model
+authors:
+  - addyosmani
+  - developit
+  - kaycebasques
+  - megginkearney
```

Comment:
```suggestion
```
So many people have touched this article and there's no single owner so I think it's fine to just remove all authors.